features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/partials/form-field.hbs

```diff
@@ -40,10 +40,11 @@
 
                 <div class="tbl-upload">
                     <div class="tbl-upload-row">
-                        <input type="text" class="txt-filepath col-lg-6 col-sm-8 col-xs-6" disabled="true" name='{{field.name}}' value='{{field.value}}'/>
+                        <input id='input_{{field.name}}' type="text" class="txt-filepath col-lg-6 col-sm-8 col-xs-6" disabled="true" name='{{field.name}}' value='{{field.value}}'/>
                         <input type="hidden" value="{{field.value}}" name='{{field.name}}'>
                         <button class="btn btn-upload btn-default">Browse</button>
-                        <input id='{{field.name}}'  accept="image/*" type='file' accept="image/*" name='{{field.name}}' class="col-lg-6 col-sm-12 col-xs-12"/>
+                        <input id='{{field.name}}'  accept="image/*" type='file' accept="image/*" name='{{field.name}}' class="col-lg-6 col-sm-12 col-xs-12"
+                               inputId='input_{{field.name}}' acceptType="Image"/>
```

Comment:
Try to use JQuery scope and find the relevant input by ID, instead of custom tags "inputId" and "acceptType"